Mathlib/Init/Data/List/Lemmas.lean

```diff
@@ -72,20 +72,25 @@ alias ⟨eq_or_mem_of_mem_cons, _⟩ := mem_cons
 #align list.mem_append_left List.mem_append_left
 #align list.mem_append_right List.mem_append_right
 
-theorem not_bex_nil (p : α → Prop) : ¬∃ x ∈ @nil α, p x := fun ⟨_, hx, _⟩ => List.not_mem_nil _ hx
-#align list.not_bex_nil List.not_bex_nil
+theorem not_exists_nil (p : α → Prop) : ¬∃ x ∈ @nil α, p x :=
```

Comment:
```suggestion
theorem not_exists_mem_nil (p : α → Prop) : ¬∃ x ∈ @nil α, p x :=
```

Reply:
This one already exists...

Reply:
AFAICT, it is not used in mathlib, though.